include/seccomp.h.in

```diff
@@ -695,6 +744,58 @@ int seccomp_export_pfc(const scmp_filter_ctx ctx, int fd);
  */
 int seccomp_export_bpf(const scmp_filter_ctx ctx, int fd);
 
+
+/**
+ * Allocate a pair of notification request/response structures.
+ * @param req the request location
+ * @param resp the response location
+ *
+ * This function allocates a pair of request/response structure by computing
+ * the correct sized based on the currently running kernel. It returns zero on
+ * success, and negative values on failure.
+ */
+int seccomp_alloc_notifications(struct seccomp_notif **req,
```

Comment:
The libseccomp naming convention tends to stick more to the "namespacing" approach (no, not that kind) and not the "noun-verb-noun" approach.  For example, instead of `seccomp_alloc_notifications()` one should use `seccomp_notifications_alloc()` or similar.

This comment applies to all the other things defined in "include/seccomp.h.in".